CHANGES

```diff
@@ -163,4 +163,8 @@ Version 2.1.15
 
 Version 2.1.16
 --------------
-* Bugfix: chimp --wait-until-done now honors the noprompt flag. 
+* Bugfix: chimp --wait-until-done now honors the noprompt flag.
+
+Version 2.1.27
```

Comment:
Your version jump is to 2.1.17 but your comment in CHANGES indicates 2.1.27.  An off by 10 error!